README.md

```diff
@@ -223,6 +223,10 @@ files after saving by using the
 have the plugin, just add `let g:syntastic_haml_checkers = ['haml_lint']` to
 your `.vimrc`.
 
+### Vim 8 / Neovim
```

Comment:
Neomake also works out of the box with it.

I wonder at the value of these paragraphs. It seems more like the responsibility of the editor library than this readme to tell you about plugins.